### DISCUSSION TITLE
MTV asia

### DISCUSSION THREAD

**Discussion by DoctorHoot927** [iptv-org/community#773] [Q&A]
### Channel Name

MTV Asia

### Channel ID

MTVAsia.sg

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
According to our database this channel was closed on 1 January 2023.

https://iptv-org.github.io/channels/sg/MTVAsia
